packages/contracts/contracts/DataTypes.sol

```diff
@@ -8,41 +8,13 @@ pragma experimental ABIEncoderV2;
 contract DataTypes {
 
     /*** Structs ***/
-    struct Range {
-        uint256 start;
-        uint256 end;
+    struct Property {
+        address decider;
+        bytes input;
     }
 
-    struct StateObject {
-        address predicateAddress;
-        bytes data;
-    }
-
-    struct StateUpdate {
-        StateObject stateObject;
-        Range range;
-        uint256 plasmaBlockNumber;
-        address depositAddress;
-    }
-
-    struct Checkpoint {
-        StateUpdate stateUpdate;
-        Range subrange;
-    }
-
-    struct Transaction {
-        address depositAddress;
-        bytes body;
-        Range range;
-    }
-
-    struct AssetTreeNode {
-        bytes32 hashValue;
-        uint256 start;
-    }
-    
-    struct StateSubtreeNode {
-        bytes32 hashValue;
-        uint128 start;
+    struct Claim {
+        uint64 numProvenContradictions;
```

Comment:
Any particular reason why this is `uint64`? It can probably be quite a bit smaller, no?